constructors.js

```diff
@@ -72,6 +105,34 @@
    * @param  {number} damage  Amount of damage to deal to the spellcaster
    */
 
+Spellcaster.prototype.inflictDamage = function (damage){
+  if (this.isAlive === true){
```

Comment:
this extra check for absolute value of `true` is good, however, since `this.isAlive` is a property that should only ever be of type `boolean`, it is more common to just use the variable as the entire boolean expression.
```js
if (this.isAlive){

}
```